### ISSUE TITLE
Error on first command!

### ISSUE THREAD

**Issue by Alfriedi** [buildcenter/WindowsBuilder#3]
Hi. First off, thank you for this project. The potential for it is huge and will be a massive help to SysAdmins everywhere.

I'm currently trying to run this on a Win 10 Ent 64bit OS but it's failing at the first command (build configure).

I've followed the steps from the walkthrough but it's failing at the 3rd step in and I'm not sure what else to try at this point. Appreciate any help or advice you can give. Set-ExecutionPolicy is set to unrestricted and I'm running CMD as an admin.

c:\WindowsBuilder>build configure
Builder 3.1.1024.0
Copyright (c) 2018 Lizoc Inc. All rights reserved.

Precondition was false, not executing task: Help

+----------+
| Precheck |
+----------+

+-------+
| Setup |
+-------+
Importing helper modules...
* PSJapson/Lizoc.PowerShell.Japson.dll
[1251:11] Error: At C:\WindowsBuilder\tools\Builder\Builder.psm1:357 char:25 +                         Die $_ 'InvokeTaskError' -NoWill +                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [<<==>>] Exception: Could not load file or assembly 'file:///C:\WindowsBuilder\tools\PSJapson\Lizoc.PowerShell.Japson.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)

c:\WindowsBuilder>

**Comment by imacks** [comment]
0x80131515 may suggest a block file issue (files downloaded from the web is blocked by default), so this may help:

```powershell
dir * -recurse -file | % { $_ | unblock-file }
```

fyi, assemblies in `tools\psjapson\*.dll` are built on .net 4.6.2, which you should already have if on win10.

**Comment by Alfriedi** [comment]
Thank you for the response Mack. I've not had a chance to test this yet but I will feedback when I have.

**Comment by imacks** [comment]
@Alfriedi i'm closing this one for now. if it's still not working, feel free to open a new issue 👍
